### ISSUE TITLE
b2 gives ERROR: rule "Copyright" unknown in module "xml".

### ISSUE THREAD

**Issue by yoonghm** [boostorg/build#418]
Hi,

  I am using boost 1.67.0 on Ubuntu.  I let the compilation run over the night.  I had forgotten to direct outputs and errors to a file. I did `b2` again.  I have these information:
```
/home/s/src/boost_1_67_0/tools/build/src/tools/types/xml.jam:12: in load
ERROR: rule "Copyright" unknown in module "xml".
/home/s/src/boost_1_67_0/tools/build/src/kernel/modules.jam:295: in import
/home/s/src/boost_1_67_0/tools/build/src/tools/types/register.jam:36: in load
/home/s/src/boost_1_67_0/tools/build/src/kernel/modules.jam:295: in import
/home/s/src/boost_1_67_0/tools/build/src/tools/stage.jam:18: in load
/home/s/src/boost_1_67_0/tools/build/src/kernel/modules.jam:295: in import
/home/s/src/boost_1_67_0/tools/build/src/tools/builtin.jam:27: in load
/home/s/src/boost_1_67_0/tools/build/src/kernel/modules.jam:295: in import
/home/s/src/boost_1_67_0/tools/build/src/build-system.jam:12: in load
/home/s/src/boost_1_67_0/tools/build/src/kernel/modules.jam:295: in import
/home/s/src/boost_1_67_0/tools/build/src/kernel/bootstrap.jam:139: in boost-build
/home/s/src/boost_1_67_0/boost-build.jam:17: in module scope
```

My `prefix` is `/home/s`.

**Comment by swatanabe** [comment]
AMDG

You're using an out-dated b2.

In Christ,
Steven Watanabe

**Comment by yoonghm** [comment]
How do I get an updated b2? The b2 is from `https://dl.bintray.com/boostorg/release/1.67.0/source/boost_1_67_0.tar.bz2`.

**Comment by swatanabe** [comment]
AMDG

Are you certain that you are using the b2 from 1.67
and not an (older) system installed b2?

In Christ,
Steven Watanabe

**Comment by yoonghm** [comment]
You are right, there is another `/usr/bin/b2`. I did not follow the Getting Started Guide exactly.  Let me try again using boost 1.69.
